token.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
What's the purpose of this file?

Reply:
This is a temporary token created by google drive API.

P.S Thanks for the detailed analysis. I'll make the necessary changes.